plugins/nginx-vhosts/commands

```diff
@@ -94,6 +95,11 @@ EOF
         echo "       Reloading nginx"
         restart_nginx
       fi
+
+      echo "-----> Generating app URLs..."
```

Comment:
Do we need the extra output? If so, you should use the helper in common for this :)